.github/workflows/c-cpp-clang.yml

```diff
@@ -0,0 +1,19 @@
+name: C/C++ CI (CLang)
+
+on:
+  push:
+    branches: [ mainline ]
+  pull_request:
+    branches: [ mainline ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
I sugguest using [jammy](https://hub.docker.com/layers/library/ubuntu/jammy/images/sha256-56887c5194fddd8db7e36ced1c16b3569d89f74c801dc8a5adbf48236fb34564?context=explore) to avoid surprises when ubuntu-latest switches to mantic.

Reply:
I guess that's actually wanted (running the latest version to have the latest compilers available).

Reply:
Yup. That's the point. Always getting latest Ubuntu tool chain.